rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -570,6 +570,9 @@ http {
     server {
         server_name {{ $server.Hostname }} {{ $server.Alias }};
 
+        {{/* default value of proxy_host is the name and port of a proxied server as specified in the proxy_pass directive */}}
+        set $proxy_host {{ $server.Hostname }};
```

Comment:
This does not make sense, `$server.Hostname` is not the server being proxied. If we want to properly set the value of this then we have to do it in Lua when we pick upstream server in balancer.lua - that is the server being proxied.

Reply:
I just checked, `proxy_host` can not be changed by Lua.

Reply:
> This does not make sense, $server.Hostname is not the server being proxied

What do you mean?
(the default value of the proxy_host variable is parsed from the proxy_pass directive)